xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -8368,10 +7996,105 @@ class CudnnExecutionPlanRunner<void(Args...)>
   mutable int64_t initial_offset_ = 0;
   int64_t offset_increment_ = 0;
   int64_t rng_seed_;
-  bool is_flash_attention_;
 };
 #endif  // CUDNN_VERSION >= 8100
 
+template <typename Sig>
```

Comment:
Is this forward declaration needed?

Reply:
It looks like this is needed. If I remove it, it shows the error `Explicit specialization of undeclared template class 'CudnnGraphRunner'` while defining it later.

Reply:
Ah, I see now. This is only implemented as specialization.